2-library/unit-testing/pom.xml

```
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.6core.lib</groupId>
+    <artifactId>unit-testing</artifactId>
```

what do you think about such a name? 
e.g. library for bundling all depencencies needed for unit tests in our project
p.s. rename directory as well if agree 
```suggestion
    <artifactId>unit-test-bundle</artifactId>
```

---

Great. It's a pity it didn't occur to me myself. Thank you!